core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-or-update-solution-modal.component.html

```diff
@@ -24,6 +24,8 @@ <h3>{{ solutionType !== null ? 'Update Solution' : 'Add Solution' }}</h3>
       <button class="md-button oppia-learner-confirm-button oppia-learner-submit-answer-button e2e-test-submit-answer-button"
               (click)="onSubmitFromSubmitButton()"
               *ngIf="data.correctAnswer === undefined && shouldAdditionalSubmitButtonBeShown()"
+              onblur="this.style.border = 'none';"
```

Comment:
Can we not handle this by using css selectors such as ‘:focus’

Reply:
Yes we can, I made the changes can you take another look please?